api/routers/main.js

```diff
@@ -1,7 +1,8 @@
 const router = require('express').Router();
+const passport = require('../policies/passport');
 const MainController = require('../controllers/main');
 
-router.get('/', MainController.index);
-router.get('/sites(/*)?', MainController.index);
+router.get('/', passport, MainController.index);
+router.get('/sites(/*)?', passport, MainController.index);
```

Comment:
I don't think you need to include the `passport` middleware here if you use `req.isAuthenticated()` as the method to check for authentication in the controller.